### ISSUE TITLE
L013 not applied if only one select target

### ISSUE THREAD

**Issue by MLux-** [sqlfluff/sqlfluff#1066]
L013 (Column expression without alias. Use explicit 'AS' clause.) is not being applied if there is only one select target. 

## Expected Behaviour
Error message for rule L013 (Column expression without alias. Use explicit 'AS' clause.) for the following SQL
```
select getdate()
from x

```

## Observed Behaviour
Lint passes without any errors.

## Steps to Reproduce
1. Create a SQL file, a.sql, like the following:
```
select getdate()
from x

```
2. Execute:
```
sqlfluff lint a.sql
```

The lint passes

## Version
```
sqlfluff --version              
sqlfluff, version 0.5.6
```
```
python --version
Python 3.8.2
```

## Configuration
No configurations set/all defaults.

**Comment by barrywhart** [comment]
I did some reading, and this behavior is:
* Deliberate
* Documented

See the `allow_scalar` flag [here](https://docs.sqlfluff.com/en/stable/rules.html#sqlfluff.core.rules.Rule_L013).

I ran the test with this `.sqlfluff` file and confirmed the warning is produced.
```
[sqlfluff:rules:L013]
allow_scalar = False
```

I think the reason for this flag and the default behavior is that single-column queries are often used to produce values that are consumed directly (e.g. nested queries), in which case, the column name is not referenced, thus irrelevant.
